dynamoose.d.ts

```diff
@@ -16,6 +16,18 @@ declare module "dynamoose" {
   export function setDefaults(options: ModelOption): void;
   export function setDDB(ddb: _AWS.DynamoDB): void;
   export function revertDDB(): void;
+  export function transaction<DataSchema, KeySchema>(
+    items: Array<
+      Promise<ModelSchema<DataSchema>>
+      | _AWS.DynamoDB.TransactWriteItem
+      | _AWS.DynamoDB.TransactGetItem
+    >,
+    options?: TransactionOptions,
+    next?: any
```

Comment:
I don't think this is correct. This is a callback, not an `any`. See the `get` example I found in this file.

```js
get(key: KeySchema, callback?: (err: Error, data: DataSchema) => void): Promise<ModelSchema<DataSchema> | undefined>;
```